### PR TITLE
#461 we need more Javadoc examples

### PR DESCRIPTION
**Pull request by neonailol**
Samples for 
- AndWithIndex
- Folded
- Max, Min, SumOf, NumberOf
- RetryScalar
- StickyScalar, SyncScalar
- Ternary

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#589](https://github.com/yegor256/cactoos/issues/589) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#589](https://github.com/yegor256/cactoos/issues/589) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/java/org/cactoos/scalar/And.java [hunk 1]
```diff
@@ -39,10 +39,13 @@
...
- * <pre> new And(
- *   new IterableOf("Mary", "John", "William", "Napkin"),
- *   name -> System.out.printf("The name: %s\n", name)
- * ).value();</pre>
+ * <pre>
+ * new And(
+ *  new ProcOf<>(input -> System.out.printf("\'%s\' ", input) ),
+ *  new IterableOf<>("Mary", "John", "William", "Napkin")
+ * ).value();
+ * // will print 'Mary' 'John' 'William' 'Napkin' to console
```
Comment (amihaiemil) [inline src/main/java/org/cactoos/scalar/And.java]: @neonailol I think "standard output" is better than "console" here :)

File: src/main/java/org/cactoos/scalar/And.java [hunk 2]
```diff
@@ -39,10 +39,13 @@
  * {@link java.util.stream.Stream#forEach(java.util.function.Consumer)}
  * works:</p>
  *
- * <pre> new And(
- *   new IterableOf("Mary", "John", "William", "Napkin"),
- *   name -> System.out.printf("The name: %s\n", name)
- * ).value();</pre>
+ * <pre>
+ * new And(
```
Comment (amihaiemil) [inline src/main/java/org/cactoos/scalar/And.java]: @neonailol what about indentation? I see in some places the indentation is of 2 spaces, in some of 4. I think we should keep it consistend, of 4 spaces.

File: src/test/java/org/cactoos/iterable/PropertiesOfTest.java [hunk 3]
```diff
@@ -29,6 +29,7 @@
 import org.cactoos.ScalarHasValue;
 import org.cactoos.func.MatcherOf;
 import org.cactoos.io.InputOf;
+import org.cactoos.io.ResourceOf;
```
Comment (amihaiemil) [inline src/test/java/org/cactoos/iterable/PropertiesOfTest.java]: @neonailol What is it with this code? The task was only about javadocs :)

**Comment by neonailol** [comment]
@amihaiemil thanks, updated my formatting to be consistent, and removed unnecessary import

**Comment by amihaiemil** [comment]
@rultor merge it pls

**Comment by neonailol** [comment]
@yegor256 Please confirm merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+240](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9045](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#589](https://github.com/yegor256/cactoos/issues/589) is now out of scope

**Pull request merged**